src/core/Header/HeaderButton.tsx

```diff
@@ -0,0 +1,104 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import '@itwin/itwinui-css/css/header.css';
+import SvgCaretDownSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretDownSmall';
+import SvgCaretUpSmall from '@itwin/itwinui-icons-react/cjs/icons/CaretUpSmall';
+
+import cx from 'classnames';
+import React from 'react';
+
+import { DropdownMenu } from '../DropdownMenu';
+import { useTheme } from '../utils/hooks/useTheme';
+
+export type HeaderButtonProps = {
+  /**
+   * Title to display.
+   */
+  title: React.ReactNode;
```

Comment:
I would go with `name` not to confuse with native html `title`

Reply:
Changed